task/arm-ttk/Expand-AzTemplate.ps1

```diff
@@ -122,7 +122,7 @@
                     $preferredJsonFile = $TemplatePath |
                         Get-ChildItem -Filter *.json |
                         # for a file named azuredeploy.json, prereq.azuredeploy.json or mainTemplate.json
-                        #Where-Object { 'azuredeploy.json', 'mainTemplate.json', 'prereq.azuredeploy.json' -contains $_.Name } |
+                        Where-Object { 'azuredeploy.json', 'mainTemplate.json', 'prereq.azuredeploy.json' -contains $_.Name } |
```

Comment:
This line needs to be commented out, as was done in the previous version. This removes the restriction that the TTK only runs a top level template.